src/jit/compiler.h

```diff
@@ -9329,6 +9331,26 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         return compRoot->m_arrayInfoMap;
     }
 
+    inline bool TryGetArrayInfo(GenTreeIndir* indir, ArrayInfo* arrayInfo)
```

Comment:
This method could use a function header explaining what it handles. (why it returns true vs. false etc...)